.github/actions/install-bpftool/action.yml

```diff
@@ -1,16 +1,41 @@
-name: 'install-bpftool'
-description: 'Install bpftool'
+name: "install-bpftool"
+description: "Install bpftool"
 
 inputs:
   version:
     description: "Version of bpftool to install (format: 'vX.Y.Z')"
     required: false
     default: "v7.3.0"
+  arch:
+    description: "Machine architecture (e.g.: 'amd64')"
+    required: true # To make it difficult for a user to rely on the wrong default.
```

Comment:
nit: you could rely on the runner architecture instead of passing it as input https://docs.github.com/en/actions/reference/workflows-and-actions/variables